### PR TITLE
Add Transfer.app v2.0

### PR DESCRIPTION
**Pull request by Drikani**
Added new cask for the TFTP-Server app Transfer

**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

_In the following questions `<cask>` is the token of the cask you're submitting._

After making all changes to a cask, verify:

- [x] The submission is for [a stable version](https://docs.brew.sh/Acceptable-Casks#stable-versions) or [documented exception](https://docs.brew.sh/Acceptable-Casks#but-there-is-no-stable-version).
- [x] `brew audit --cask <cask>` is error-free.
- [x] `brew style --fix <cask>` reports no offenses.

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference](https://docs.brew.sh/Cask-Cookbook#token-reference).
- [x] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues).
- [x] Checked the cask is submitted to [the correct repo](https://docs.brew.sh/Acceptable-Casks#finding-a-home-for-your-cask).
- [x] `brew audit --new-cask <cask>` worked successfully.
- [x] `brew install --cask <cask>` worked successfully.
- [x] `brew uninstall --cask <cask>` worked successfully.

### CODE REVIEW

**Comment by SMillerDev** [comment]
Please add a [`zap` stanza](https://docs.brew.sh/Cask-Cookbook#stanza-zap) here. The [CreateZap](https://github.com/nrlquaker/homebrew-createzap) tool can help you with this.

File: Casks/transfer.rb [hunk 1]
```diff
@@ -4,10 +4,19 @@
...
 
   depends_on macos: ">= :mojave"
 
   app "Transfer.app"
+
+  zap trash: [
+  "~/Library/Application Support/Transfer",
+  "~/Library/Caches/com.adriangranados.Transfer",
+  "~/Library/Caches/com.apple.helpd/Generated/com.intuitibits.transfer.help*2.0",
+  "~/Library/Group Containers/com.apple.MessagesLegacyTransferArchive",
```
Comment (SMillerDev) [inline Casks/transfer.rb:18]: This doesn't seem right? Are you sure these files are generated by the app?

Comment (Drikani) [reply Casks/transfer.rb]: Yeah I dont't know what happened there...
I fixed the zap block and also re-did the styling.
I hope everything is now okay?

**Comment by miccal** [comment]
Thank you @Drikani.

**Pull request merged**